src/main/java/uk/gov/pay/api/model/CreatePaymentResult.java

```diff
@@ -63,21 +73,25 @@
 
     @JsonProperty
     @ApiModelProperty(name = LINKS_JSON_ATTRIBUTE)
+    @Schema(name = LINKS_JSON_ATTRIBUTE)
     private PaymentLinks links;
 
     @JsonProperty
     @ApiModelProperty(name = "provider_id", example = "null")
+    @Schema(name = "provider_id", example = "null")
     private String providerId;
 
     @JsonProperty
     @ApiModelProperty(name = "metadata", dataType = "Map[String,String]")
+    @Schema(name = "metadata")
     private ExternalMetadata metadata;
 
     @JsonProperty
     @ApiModelProperty(name = "email", example = "citizen@example.org", required = false)
+    @Schema(name = "email", example = "citizen@example.org")
     private String email;
 
     @JsonProperty(value = "card_details")
-    @ApiModelProperty(name = "card_details", required = false)
```

Comment:
Is it accidental or purposeful removal?

Reply:
Good spot, replaced with `@Schema instead of adding and updated